api/messaging/spec/webhooks_public.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Not sure I understand why we're splitting this into a new file? This breaks old patterns.

Reply:
These are public routes, others are not. In one file, all the routes may be either public or private.